sympy/ntheory/primetest.py

```diff
@@ -110,6 +110,19 @@ def mr(n, bases):
     return True
 
 
+def isprimeE(n,b):
+	"""isprimeE(n) - Test whether n is prime or an Euler pseudoprime to base b."""
```

Comment:
docstrings usually do not repeat the name of the function. """Test whether n is prime or an Euler pseudoprime to base b.""" would be enough.

Reply:
Please do comment unnecessarily on such trivial things if there is any conceptual/coding mistake then only comment